src/olympia/hero/admin.py

```diff
@@ -1,16 +1,47 @@
+from django import forms
 from django.contrib import admin
 from django.core.exceptions import ValidationError
 from django.forms import BaseInlineFormSet
+from django.utils.safestring import mark_safe
 
-from .models import PrimaryHero, SecondaryHeroModule
+from .models import (
+    PrimaryHero, SecondaryHeroModule,
+    PrimaryHeroImage, ImageChoiceField)
 
 
 class PrimaryHeroInline(admin.StackedInline):
     model = PrimaryHero
-    fields = ('image', 'gradient_color', 'is_external', 'enabled')
+    fields = (
+        'select_image',
```

Comment:
until we're ready to completely drop `image` we'll need to leave it in here (when we drop `image` the field will need removing from the model, and the serializer updating so `select_image` is served via the API)